MapboxVision/Services/ModelPerformance.swift

```diff
@@ -19,7 +25,16 @@ public enum ModelPerformanceMode {
 /**
  Enumeration which determines performance rate of the specific model. These are high-level settings that translates into adjustment of FPS for ML model inference.
  */
-public enum ModelPerformanceRate {
+public enum ModelPerformanceRate: Comparable {
+    /// :nodoc:
+    public static func < (lhs: ModelPerformanceRate, rhs: ModelPerformanceRate) -> Bool {
+        let rates: [ModelPerformanceRate] = [.off, .low, .medium, .high]
```

Comment:
What do you think about `CaseIterable`?

We can rewrite the code below as:
```
let rates = ModelPerformanceRate.allCases
...
```

to avoid supporting `rates` constant.

Reply:
Thanks, applied.